assets/js/main.js

```diff
@@ -1,57 +1,27 @@
 $(document).ready(function() {
-  var $userMenu = $('.user-menu'),
-    $userMenuOpen = $('span.user-avatar'),
-    $menuMainItem = $('.menu-main .submenu-item'),
-    $submenuItem = $('.submenu-item .sub-menu li a'),
-    $subMenu = $('.submenu-item .sub-menu'),
-    $headerMain = $('.header-main');
 
-  $('body').mouseup(function(e) {
-    if (!$userMenu.is(e.target) && !$userMenuOpen.is(e.target) && $userMenu.has(e.target).length === 0) {
-      $userMenu.removeClass('open');
-    }
-    if (!$subMenu.is(e.target) && $subMenu.has(e.target).length === 0) {
-      $subMenu.removeClass('open');
+  $('body').mousemove(function(e) {
+    
+    if (!$('.submenu-item .sub-menu').is(e.target) && $('.submenu-item .sub-menu').has(e.target).length === 0) {
       $('.header-main').removeClass('header-open');
-    }
-  });
+    } 
 
-  $menuMainItem.hover(function() {
     if ($('.menu-main .sub-menu').is(':visible')) {
-      $headerMain.addClass('header-open');
-    } else {
-      $headerMain.removeClass('header-open');
+      $('.header-main').addClass('header-open');
     }
-  });
 
-  $menuMainItem.click(function() {
-    $('.menu-main .submenu-item > a').removeClass('active');
-    $('.sub-menu').removeClass('open');
-    $(this).find('> a').addClass('active');
-    $headerMain.addClass('header-open');
-    $(this).find('.sub-menu').addClass('open');
-  });
-
-  $('.menu-main li a').click(function() {
-    $('.menu-main li a').removeClass('active');
-    $(this).addClass('active');
   });
 
   $('#user-menu--open').click(function() {
```

Comment:
I think that it is not a good idea defining event handlers on _mouseMove_ on _body_.

Did you consider using _on_ jquery method?

Reply:
on what? need a event on body. What event do you thinks works likes mouseMove ?